### PR TITLE
Add test KeyCreate

### PR DESCRIPTION
**Pull request by mariiatuzovska**

### CODE REVIEW

File: pkg/cmd/cli/key_test.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	"optable-pair-cli/pkg/keys"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestKeyCreate(t *testing.T) {
+	t.Parallel()
+
+	tmpDir := os.TempDir()
+	keyConfigFile := tmpDir + "/test_config.json"
```
Comment (amanjpro) [inline pkg/cmd/cli/key_test.go:18]: Maybe use os.TempFile?

Comment (amanjpro) [reply pkg/cmd/cli/key_test.go:18]: Eventhough /tmp files are automatically deleted once the computer is restarted, you should still delete the "extraneous" files that you have created after the test is done

os.TempFile(...)
defer os.RemoveFile() // or whatever it is

**Review by amanjpro — COMMENTED** [review]
Can you also test that without passing force, it won't override an existing key (change the behaviour of the key-craete cmd if need be)

File: pkg/cmd/cli/key_test.go (hunk 1 above)
Comment (mariiatuzovska) [reply pkg/cmd/cli/key_test.go:18]: CreateTemp creates a new temporary file in the directory dir, opens the file for reading and writing, and returns the resulting file. I prefer to test if its created.

Comment (amanjpro) [reply pkg/cmd/cli/key_test.go:18]: Ah, ok

You still need to delete the temp dir after it is created, though

**Review by amanjpro — APPROVED** [review]

**Pull request merged**